src/coreclr/scripts/superpmi_collect_setup.py

```diff
@@ -509,9 +546,7 @@ def make_readable(folder_name):
         #     print('Copying {} -> {}'.format(dotnet_src_directory, dotnet_dst_directory))
         #     copy_directory(dotnet_src_directory, dotnet_dst_directory, verbose_output=False)
 
-        # payload
-        pmiassemblies_directory = os.path.join(workitem_directory, "pmiAssembliesDirectory")
-        input_artifacts = os.path.join(pmiassemblies_directory, coreclr_args.collection_name)
+        input_artifacts = os.path.join(workitem_payload_directory, "collectAssembliesDirectory", coreclr_args.collection_name)
```

Comment:
Consider setting variable for `"collectAssembliesDirectory"` in yml which can be used in python script as well as `proj` file.